From 86e9476cbd9c9bbc54c47a710b1354257b23ab08 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 16 May 2024 10:47:31 +0200
Subject: [PATCH 595/841] feat(stm32mp2-fdts): remove the pins-are-numbered
 property from stm32mp251

Remove the deprecated property "pins-are-numbered" from pinctrl and
pinctrl_z nodes of stm32mp25 soc to ensure to be conform with the
upstream series of the link below.

Link: https://patchwork.kernel.org/project/linux-arm-kernel/list/?series=69786

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 8a3b9f06bdaa880e7f1e3d37feaa3fe3a59f8511 feat(stm32mp2-fdts): remove the pins-are-numbered property from stm32mp251]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I1ed98c94c5003bc9903229957cb072da4211238f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/380111
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp251.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 274cd12f4..e0c88faa8 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -517,7 +517,6 @@
 			#size-cells = <1>;
 			compatible = "st,stm32mp257-pinctrl";
 			ranges = <0 0x44240000 0xa0400>;
-			pins-are-numbered;
 
 			gpioa: gpio@44240000 {
 				gpio-controller;
@@ -646,7 +645,6 @@
 			#size-cells = <1>;
 			compatible = "st,stm32mp257-z-pinctrl";
 			ranges = <0 0x46200000 0x400>;
-			pins-are-numbered;
 
 			gpioz: gpio@46200000 {
 				gpio-controller;
-- 
2.47.2

